lib/auth.js

```diff
@@ -561,241 +341,6 @@ api.setEmail = function(actor, id, email, callback) {
-      changes.sysPasscode = results.generatePasscode;
-      database.collections.identity.update(
-        {id: database.hash(identity.id)},
-        {$set: database.buildUpdate(changes, 'identity')},
-        database.writeOptions,
-        function(err, result) {
-          if(err) {
-            return callback(err);
-          }
-          if(result.result.n === 0) {
-            return callback(new BedrockError(
-              'Could not set Identity password. Identity not found.',
-              'NotFound'));
-          }
-          callback();
-        });
-    }]
-  }, function(err) {
-    callback(err, changes);
-  });
-};
-
-/**
- * Verifies the Identity's password against the stored password.
- *
- * @param identity the Identity with the password to verify.
- * @param callback(err, verified) called once the operation completes.
- */
-api.verifyPassword = function(identity, callback) {
-  _verifyPasswordHash(identity, 'password', callback);
-};
-
-/**
- * Verifies the Identity's passcode against the stored passcode.
- *
- * @param identity the Identity with the passcode to verify.
- * @param callback(err, verified) called once the operation completes.
- */
-api.verifyPasscode = function(identity, callback) {
-  _verifyPasswordHash(identity, 'passcode', callback);
-};
-
-/**
- * Verifies the Identity's passcode against the stored passcode and sets
- * the Identity's email address as verified upon success.
- *
- * @param actor the Identity performing the action.
- * @param identity the Identity with the passcode to verify.
- * @param callback(err, verified) called once the operation completes.
- */
-api.verifyEmail = function(actor, identity, callback) {
-  async.auto({
-    checkPermission: function(callback) {
-      brIdentity.checkPermission(
-        actor, PERMISSIONS.IDENTITY_EDIT, {resource: identity}, callback);
-    },
-    verifyPasscode: ['checkPermission', function(callback) {
-      _verifyPasswordHash(identity, 'passcode', callback);
-    }],
-    update: ['verifyPasscode', function(callback, results) {
-      if(!results.verifyPasscode) {
-        return callback();
-      }
-      database.collections.identity.update(
-        {id: database.hash(identity.id)},
-        {$set: {'identity.sysEmailVerified': true}},
-        database.writeOptions, callback);
-    }]
-  }, function(err, results) {
-    callback(err, results.verifyPasscode);
-  });
-};
-
-/**
- * Sends an Identity's or multiple Identity's passcodes to their contact point
- * (eg: email address). The Identities must all have the same contact point and
- * must be populated.
- *
- * @param identities the Identities to send the passcode to.
- * @param usage 'reset' if the passcode is for resetting a password,
- *          'verify' if it is for verifying an email address/contact point.
- * @param callback(err) called once the operation completes.
- */
-api.sendPasscodes = function(identities, usage, callback) {
```

Comment:
This is now in `bedrock-authn-password` -- let's mark it deprecated and call into that module and make this a simple alias.